packages/clay-color-picker/src/Editor.tsx

```diff
@@ -153,8 +156,32 @@ export function Editor({
 		[b, 'b'],
 	];
 
+	const [alpha, setAlpha] = useState<number>(1);
+
+	React.useEffect(() => {
+		let currentAlpha = color.getAlpha();
+
+		if (currentAlpha !== 0 && currentAlpha !== 1) {
+			currentAlpha = +currentAlpha.toFixed(2);
+		}
+
+		setAlpha(currentAlpha);
+	}, [alpha, color]);
+
 	return (
 		<>
+			<Hue
+				onChange={(hue) => {
+					if (hue < LimitValue.min) {
+						hue = LimitValue.min;
+					} else if (hue > LimitValue.maxHue) {
+						hue = LimitValue.maxHue;
+					}
+					onHueChange(hue);
+					onColorChange(tinycolor({h: hue, s, v}).setAlpha(alpha));
```

Comment:
I guess to avoid having an internal `alpha` state in the Editor, here you can just get the alpha from the color.

```suggestion
					onColorChange(tinycolor({h: hue, s, v}).setAlpha(color.getAlpha()));
```

Reply:
Fixed